			<div class="nice_margins">
				<div class="row-fluid">
					<div class="span4">
						<div class="row-fluid">
							<div class="span2 bottom-padding"></div>
							<div class="span8 bottom-padding">
								<img src="/assets/img/sumac.jpg" class="img-rounded">
                                <br><br><br><br>
							</div>
							<div class="span2 bottom-padding"></div>
						</div>
						<div class="row-fluid">
							<div class="span2 bottom-padding"></div>
							<div class="span8 bottom-padding">
                                <img src="/assets/img/MSDS.jpg" class="img-rounded">
							</div>
							<div class="span2 bottom-padding"></div>
						</div>
					</div>
					<div class="span8">
						<h4>SUMAC 2.0</h4>
                        <p>8/2016: SUMAC version 2.0 is <i>significantly</i> faster than previous versions due to the
                        use of a new clustering algorithm.</p>
                        <p>Supermatrix Constructor (SUMAC) is a tool to data-mine GenBank, construct phylogenetic supermatrices,
                        and assess the decisiveness of a matrix given the pattern of missing sequence data.
                        SUMAC calculates a novel metric, Missing Sequence Decisiveness Scores (MSDS), which measure how
                        much each individual missing sequence contributes to the decisiveness
                        of the matrix. MSDS can be used to compare supermatrices and prioritize the acquisition of new sequence data.
                        </p><p>
                        SUMAC constructs supermatrices either through an exploratory clustering of all GenBank sequences within a taxonomic group,
                        or by using guide sequences to build homologous clusters in a more targeted manner.
                        SUMAC will assemble supermatrices for any taxonomic group recognized in GenBank, and is optimized to run
                        on multicore processors by utilizing multiple parallel processes.
                        SUMAC is implemented as a Python package that can run as a stand-alone command line program,
                        or its modules and objects can be incorporated within other programs.
                        </p>
                        <p>SUMAC works on Linux/OSX (not MS Windows), and is available at <a href="https://github.com/wf8/sumac">https://github.com/wf8/sumac</a> 
                        under the open source <a href="http://www.opensource.org/licenses/gpl-3.0.html">GPLv3 license</a>.
                        </p>
                        <p><b>Citation:</b><br>
                        Freyman, W.A. 2015. SUMAC: constructing phylogenetic supermatrices and assessing partially decisive taxon coverage. <em>Evolutionary Bioinformatics</em> 2015:11 263-266
                           <a href="http://www.la-press.com/sumac-constructing-phylogenetic-supermatrices-and-assessing-partially--article-a5232">[html]</a>
                           <a href="http://www.la-press.com/redirect_file.php?fileId=6991&filename=5232-EBO-SUMAC:-Constructing-Phylogenetic-Supermatrices-and-Assessing-Partially.pdf&fileType=pdf">[pdf]</a></p>
					</div>
				</div>
			</div>
      		<hr>
